### ISSUE TITLE
Models named Query cause an internal GraphQL Parse Error

### ISSUE THREAD

**Issue by dkantereivin** [prisma/prisma#8153]
### Bug description

When naming models any case of `query`, no operations can be performed. Originally discovered on an actual project of mine, and discussed with @janpio. Initial suspicion was that this was due to the actual content of my query, but once I managed to troubleshoot the begun, the name of the variable (which fixed the problem) became my theory.

Further confirmed by an extremely simple model with just one field producing the exact same error.

### How to reproduce

Setup a basic prisma project using NPM. This means the standard:
```
npm init
npm i -D typescript ts-node @types/node prisma
npx prisma init
npx prisma migrate
```
Just use a simple sqlite3 database, it won't even query the DB because it'll crash first.

### Expected behavior

This should work, and not cause this query error, or there should at least be a more descriptive error message which is specific to this error and indicates the troubleshooting step. The standard error (below) is totally useless.
```
C:\Users\david\Documents\Dev\prisma-querybug\node_modules\@prisma\client\runtime\index.js:33900
          throw new import_engine_core.PrismaClientUnknownRequestError(message, this.prisma._clientVersion);
                ^
Error:
Invalid `prisma.query.create()` invocation:


  Error parsing GraphQL query: query parse error: Parse error at 6:3
Unexpected `}[Punctuator]`
Expected `Name`
```

If the decision is made not to correct this behaviour, this should be documented.

### Prisma information

Prisma Schema:
```
datasource db {
  provider = "sqlite"
  url      = "file:./test.db"
}

generator client {
  provider = "prisma-client-js"
}

model Query {
  id String @id @default(uuid())
  // literally not including anything else, its not because of the model contents...
}
```

Test Code:
```ts
import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()

async function main() {
    prisma.$connect();
    const qry = await prisma.query.create({
        data: {}
    });

    console.log(qry);
}

main();
```
with package.json:
```
  "devDependencies": {
    "@types/node": "^16.0.1",
    "prisma": "^2.26.0",
    "ts-node": "^10.0.0",
    "typescript": "^4.3.5"
  },
  "dependencies": {
    "@prisma/client": "^2.26.0"
  }
```

### Environment & setup

- OS:  Windows, and likely all.
- Database: PostgreSQL and SQLite verified, and likely all.
- Node.js version:  15.3.0

### Prisma Version

```
prisma               : 2.26.0
@prisma/client       : 2.26.0
Current platform     : windows
Query Engine         : query-engine 9b816b3aa13cc270074f172f30d6eda8a8ce867d (at node_modules\@prisma\engines\query-engine-windows.exe)
Migration Engine     : migration-engine-cli 9b816b3aa13cc270074f172f30d6eda8a8ce867d (at node_modules\@prisma\engines\migration-engine-windows.exe)
Introspection Engine : introspection-core 9b816b3aa13cc270074f172f30d6eda8a8ce867d (at node_modules\@prisma\engines\introspection-engine-windows.exe)
Format Binary        : prisma-fmt 9b816b3aa13cc270074f172f30d6eda8a8ce867d (at node_modules\@prisma\engines\prisma-fmt-windows.exe)
Default Engines Hash : 9b816b3aa13cc270074f172f30d6eda8a8ce867d
Studio               : 0.408.0
```

**Comment by pantharshit00** [comment]
I can confirm this bug. This is can be part of https://github.com/prisma/prisma/issues/7963

**Comment by matthewmueller** [comment]
Added it to https://github.com/prisma/prisma/issues/7963, thanks @pantharshit00!

**Comment by SevInf** [comment]
This issue is fixed and will be available in Prisma 5.1
